app/templates/components/wds-global-navigation.hbs

```diff
@@ -37,15 +33,19 @@
 	{{/if}}
 	<div class="wds-global-navigation__start-a-wiki{{if model.user ' wds-user-is-logged-in'}}">
 		{{#wds-button secondary=true href=model.create_wiki.header.href class="wds-global-navigation__start-a-wiki-button"}}
-			<span class="wds-global-navigation__start-a-wiki-caption">{{i18n model.create_wiki.header.title.key}}</span>
-			{{wds-icon name='plus' class='wds-global-navigation__start-a-wiki-icon'}}
+			{{i18n model.create_wiki.header.title.key}}
```

Comment:
@latata we can change it later when I'm done with i18n task, but the key should be prefixed with `design-system` namespace

Reply:
yes, let's change them later when i18n thing will be ready